src/main/java/com/brandon3055/draconicevolution/common/items/tools/Magnet.java

```diff
@@ -126,19 +155,55 @@ public void onUpdate(ItemStack stack, World world, Entity entity, int slot, bool
                                         == item.getEntityItem().getItemDamage())) {
                     continue;
                 }
-                playSound = true;
 
-                if (item.delayBeforeCanPickup > 0) {
+                doMove = false;
+                dw = item.getDataWatcher();
```

Comment:
I don't think it's a good idea to use the data watcher, if another mod is using the same watcher index for another thing, your code will not work or worse it will crash the game